### DISCUSSION TITLE
Partially upgrading to git upstream

### DISCUSSION THREAD

**Discussion by ghost** [lxqt/lxqt#2263] [Q&A] [answered]
On Arch Linux I downloaded `lxqt-config-git`'s `PKGBUILD` and edited it to use the release versions of its dependencies.

Original `PKGBUILD`: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=lxqt-config-git
Edited lines:
```
depends=("liblxqt" "libkscreen" "libxcursor" "qt5-svg" "hicolor-icon-theme")
makedepends=("git" "cmake" "qt5-tools" "lxqt-build-tools" "xf86-input-libinput")
```

I'm wondering if this is an alright thing to do, or can cause many undefined bugs.

**Comment by tsujan** [comment, marked as answer]
> ... or can cause many undefined bugs.

Yes, it can.

Before using a git LXQt component, you need to have the git versions of its LXQt dependencies; otherwise, problems will be imminent, even if the compilation is successful (by chance).
